### PR TITLE
feat: include service name within the schema check

### PR DESCRIPTION
**Pull request by n1ru4l**
closes https://github.com/kamilkisiela/graphql-hive/issues/750

### CODE REVIEW

**Comment by kamilkisiela** [comment]
It's a breaking change if somebody has the check marked as "required" on Github

File: packages/services/api/src/modules/schema/providers/schema-publisher.ts [hunk 1]
```diff
@@ -167,7 +167,7 @@ export class SchemaPublisher {
         }
 
         await this.gitHubIntegrationManager.createCheckRun({
-          name: 'GraphQL Hive - schema:check',
+          name: 'GraphQL Hive - schema:check' + input.service ? `: ${input.service}` : '',
```
Comment (klippx) [inline packages/services/api/src/modules/schema/providers/schema-publisher.ts:170]: Would it be possible to have target as well?

**Comment by n1ru4l** [comment]
> It's a breaking change if somebody has the check marked as "required" on Github
What strategy do you propose here? Unfortunately, we cannot check who marked it as required.

We could just tell users that the name will change. (low effort by both parties involved)
We could make it backward compatible (opt-in). (more effort on our side; no effort for existing users)

**Comment by dotansimha** [comment]
@n1ru4l I rebased the PR to latest, CI should pass now. 
I think the concern about the breaking changes is valid, but the impact is actually low.
@kamilkisiela what should be the next step?

**Pull request merged**